src/mapping/config/MappingConfiguration.cpp

```diff
@@ -337,6 +364,10 @@ void MappingConfiguration::xmlTagCallback(
       delete[] arg;
 
       mapping.mapping = getRBFMapping<RBFBackend::PETSc>(basisFunction, constraintValue, mapping.fromMesh->getDimensions(), supportRadius, shapeParameter, _rbfConfig.deadAxis, _rbfConfig.solverRtol, _rbfConfig.polynomial, _rbfConfig.preallocation);
+
+#elif !defined(PRECICE_NO_GINKGO)
+      mapping.mapping = getRBFMapping<RBFBackend::Ginkgo>(basisFunction, constraintValue, mapping.fromMesh->getDimensions(), supportRadius, shapeParameter, _rbfConfig.deadAxis, _rbfConfig.polynomial, false, _ginkgoParameter);
```

Comment:
This needs to go into a separate if branch. If one builds preCICE with both, PETSc and ginkgo, it will override the mapping and initialize petsc. We could potentially discuss a bit about the config interface at the next coding days.

Reply:
Also thought about that. Currently, PETSc is the preferred variant.